Dockerfile.test.gpu

```diff
@@ -55,6 +55,9 @@ RUN wget --progress=dot:mega https://bootstrap.pypa.io/get-pip.py && python get-
 RUN pip install -U --force pip setuptools requests pytest mock pytest-forked parameterized
 RUN echo pytest -v --capture=no --continue-on-collection-errors --junit-xml=/artifacts/junit.\$1.\${HOROVOD_RANK:-\${OMPI_COMM_WORLD_RANK:-\${PMI_RANK}}}.\$2.xml \${@:2} > /pytest.sh
 RUN cat /pytest.sh
+RUN echo pytest -v --capture=no --continue-on-collection-errors --junit-xml=/artifacts/junit.\$1.standalone.\$2.xml \${@:2} > /pytest_standalone.sh
+RUN chmod a+x /pytest_standalone.sh
+RUN cat /pytest_standalone.sh
```

Comment:
the cat line (here and three lines above) can go away once we are happy with the content of the script file, which we already are I think